usaspending_api/etl/tests/integration/test_load_transactions_in_delta.py

```diff
@@ -0,0 +1,1264 @@
+            update_date=datetime(year=2022, month=10, day=31),
+            generated_unique_award_id="award_procure_0004",
+            is_fpds=True,
+        )
+        baker.make(
+            "awards.TransactionNormalized",
+            id=11,
+            award=award,
+            transaction_unique_id="award_procure_0004_trans_0001",
+            update_date=datetime(year=2022, month=10, day=31),
+            is_fpds=True,
+            unique_award_key=None,
+        )
+
+        load_initial_delta_tables(spark, s3_unittest_data_bucket)
+
+        with raises(ValueError, match="Found 1 NULL in 'unique_award_key' in table raw.transaction_normalized!"):
+            call_command("load_transactions_in_delta", "--etl-level", "initial_run")
+
+    @mark.django_db(transaction=True)
+    def test_multiple_nulls_in_trans_norm_unique_award_key_from_pg(
+        self, spark, s3_unittest_data_bucket, hive_unittest_metastore_db, populate_initial_postgres_data
+    ):
+        baker.make(
+            "transactions.SourceProcurementTransaction",
+            detached_award_procurement_id=6,
+            detached_award_proc_unique="award_procure_0004_trans_0001",
+            updated_at=datetime(year=2022, month=10, day=31),
+            unique_award_key="award_procure_0004",
+        )
+        baker.make(
+            "transactions.SourceProcurementTransaction",
+            detached_award_procurement_id=7,
+            detached_award_proc_unique="award_procure_0004_trans_0002",
+            updated_at=datetime(year=2022, month=10, day=31),
+            unique_award_key="award_procure_0004",
+        )
+        award = baker.make(
+            "awards.Award",
+            id=7,
+            update_date=datetime(year=2022, month=10, day=31),
+            generated_unique_award_id="award_procure_0004",
+            is_fpds=True,
+        )
+        baker.make(
+            "awards.TransactionNormalized",
+            id=11,
+            award=award,
+            transaction_unique_id="award_procure_0004_trans_0001",
+            update_date=datetime(year=2022, month=10, day=31),
+            is_fpds=True,
+            unique_award_key=None,
+        )
+        baker.make(
+            "awards.TransactionNormalized",
+            id=12,
+            award=award,
+            transaction_unique_id="award_procure_0004_trans_0002",
+            update_date=datetime(year=2022, month=10, day=31),
+            is_fpds=True,
+            unique_award_key=None,
+        )
+
+        load_initial_delta_tables(spark, s3_unittest_data_bucket)
+
+        with raises(ValueError, match="Found 2 NULLs in 'unique_award_key' in table raw.transaction_normalized!"):
+            call_command("load_transactions_in_delta", "--etl-level", "initial_run")
+
+    @mark.django_db(transaction=True)
+    def test_one_null_in_trans_norm_unique_award_key_from_delta(
+        self, spark, s3_unittest_data_bucket, hive_unittest_metastore_db, populate_initial_postgres_data
+    ):
```

Comment:
Can tests like this be combined with the test below to cut back on Spark jobs? Will any functionality be missed if doing so?